samples/utilities/Utilities.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Azure.ResourceManager.Network" Version="1.0.0-preview.1" />
     <PackageReference Include="Azure.ResourceManager.Resources" Version="1.0.0-preview.1" />
     <PackageReference Include="Azure.ResourceManager.Storage" Version="1.0.0-preview.1" />
```

Comment:
lets review, update and cleanup any unused references

Reply:
This is the Utility for all the samples. Needs to be updated, but in another PR.